build/dependencies.props

```diff
@@ -6,6 +6,7 @@
     <JsonNetVersion>10.0.1</JsonNetVersion>
     <MicrosoftDotNetPlatformAbstractionsVersion>2.0.0</MicrosoftDotNetPlatformAbstractionsVersion>
     <MicrosoftNETCoreApp20PackageVersion>2.0.0</MicrosoftNETCoreApp20PackageVersion>
+    <MicrosoftNETCoreApp21PackageVersion>2.1.0-preview1-25907-02</MicrosoftNETCoreApp21PackageVersion>
```

Comment:
Can somebody with more repository expertise educate me on how we pick this version? The CI is failing saying it can't find it. I just copied this from Kestrel.

/cc @natemcmaster @ryanbrandenburg

Reply:
It's the most recent version on dotnet-core feed. The version from https://github.com/aspnet/Universe/blob/dev/build/dependencies.props#L32 is meant to flow to these repos. My guess is BuildTools doesn't reference aspnetcore-ci-dev which is why it doesn't find this. Perhaps it needs to reference it \ `dotnet-core`

Reply:
This repo intentionally does not reference any aspnetcore myget feeds. We removed it to ensure we avoid cyclical dependences between runtime and tools. Adding the dotnet-core myget feed is the better solution.

Reply:
Ok I added dotnet-core to the list of feeds. Lets see what fails next 😄